latest/docker-initialize.py

```diff
@@ -150,28 +150,33 @@ def buildout(self):
 
         sources = self.env.get("SOURCES", "").strip().split(",")
 
+        password = self.env.get("PASSWORD", "").strip()
+
         # If profiles not provided. Install ADDONS :default profiles
         if not profiles:
             for egg in eggs:
                 base = egg.split("=")[0]
                 profiles.append("%s:default" % base)
 
-        enabled = bool(site)
-        if not (eggs or zcml or develop or enabled):
+        if not (eggs or zcml or develop or site or password):
             return
 
         buildout = BUILDOUT_TEMPLATE.format(
+            password=password or "admin",
             findlinks="\n\t".join(findlinks),
             eggs="\n\t".join(eggs),
             zcml="\n\t".join(zcml),
             develop="\n\t".join(develop),
-            profiles="\n\t".join(profiles),
             versions="\n".join(versions),
             sources="\n".join(sources),
-            site=site or "senaite",
-            enabled=enabled,
         )
 
+        if site:
```

Comment:
from my perspective it is surprising if site config is included even though no site (URL) was specified

also what benefits do we have from a site config with default URL which is not `enabled`?